test/contrib/s3_test.py

```diff
@@ -527,3 +511,11 @@ def _run_multipart_test(self, part_size, file_size, **kwargs):
         key_size = s3_client.get_key(s3_path).size
         self.assertEqual(file_size, key_size)
         tmp_file.close()
+
+    def _run_copy_response_test(self, response):
```

Comment:
I'd find the way to check that the size is strictly equal to the sum instead of using GreaterEqual. Same thing to the number of items.